src/components/data/PathListAutomation.js

```diff
@@ -193,246 +201,295 @@ export default function PathListAutomation(props) {
-                pattern: "",
-                foldersOnly: false,
-            }}
-        >
-            {({ handleSubmit }) => {
-                return (
-                    <Form>
-                        {status === constants.LOADING && (
-                            <CircularProgress
-                                size={30}
-                                thickness={5}
-                                style={{
-                                    position: "absolute",
-                                    top: "50%",
-                                    left: "50%",
-                                }}
-                            />
-                        )}
-
-                        <Grid
-                            container
-                            direction="column"
-                            justify="center"
-                            alignItems="stretch"
-                            spacing={1}
-                        >
-                            {createPathListError && (
-                                <Grid item xs>
-                                    <ErrorTypographyWithDialog
-                                        baseId={baseId}
-                                        errorMessage={errorMsg}
-                                        errorObject={createPathListError}
-                                    />
-                                </Grid>
-                            )}
-                            <Grid item xs>
-                                <Typography variant="body2">
-                                    {t("pathListInputLbl")}
-                                </Typography>
-                                {infoTypeError && (
-                                    <ErrorTypographyWithDialog
-                                        baseId={baseId}
-                                        errorMessage={t("infoTypeFetchError")}
-                                        errorObject={infoTypeError}
-                                    />
-                                )}
-                                <Field
-                                    id={buildId(
-                                        baseId,
-                                        ids.PATH_LIST_AUTO_INPUTS
-                                    )}
-                                    name="selectedPaths"
-                                    required={true}
-                                    component={MultiInputSelector}
-                                    height="25vh"
-                                    label={t("suggestionSelection_any_plural")}
-                                />
-                            </Grid>
-                            <Grid item xs>
-                                <Field
-                                    id={buildId(
-                                        baseId,
-                                        ids.PATH_LIST_AUTO_FOLDERS_ONLY_SWITCH
+        <>
+            <DEDialog
```

Comment:
I haven't done a deep review, but 3 things:
* If you switch to using DEDialog, then you cannot use `DialogContent` or `DialogActions` directly.  
  * You can see the component [here](https://github.com/cyverse-de/sonora/blob/master/src/components/utils/DEDialog.js). 
  * It's already using `DialogContent` and `DialogActions` for you, as well as setting up `DialogTitle`. 

* If you recall from our zoom call, we were looking at [this](https://github.com/cyverse-de/sonora/blob/master/src/components/data/CreateFolderDialog.js#L77:L90) as an example to follow.  Formik, then Form, then Dialog. OR if you want to continue to use `DEDialog`, then [here's](https://github.com/cyverse-de/sonora/blob/master/src/components/doi/UpdateRequestDialog.js#L88:L105) a similar example.  Formik, then Form, then DEDialog.

* If you've set up the `Dialog` or `DEDialog` correctly, you don't need to add anything else to get it to scroll.  It will scroll automatically by default.

Hope this helps!

Reply:
I've replaced the appropriate components but it still does not show this error message when `Done` is clicked although it highlights it red since its marked as required. 
 
![Screen Shot 2021-04-23 at 11 02 21 AM](https://user-images.githubusercontent.com/55575102/115911859-7a717a80-a423-11eb-8456-ba60efba7b65.png)

Reply:
As pointed out by Ashley this message comes from the browser: https://stackoverflow.com/questions/5392882/why-is-chrome-showing-a-please-fill-out-this-field-tooltip-on-empty-fields